jax/api.py

```diff
@@ -107,8 +107,8 @@ def jit(fun, static_argnums=(), device=None, backend=None):
       Optional, the Device the jitted function will run on. (Available devices
       can be retrieved via ``jax.devices()``.) The default is inherited from
       XLA's DeviceAssignment logic and is usually to use ``jax.devices()[0]``.
-    backend: This is an experimental feature and the API is likely to change.
-      Optional, a string representing the xla backend. 'cpu','gpu', or 'tpu'.
+      Can also be passed ``'cpu'``, ``'gpu'``, or ``'tpu'`` to use the default
```

Comment:
It does not seem nice to overload the device argument with a string denoting the platform. I assume this would be rarely needed, why not ask the user to use `devices=devices("gpu")[0]`?